src/Peering/Peering/ChangeLog.md

```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-
+## Version 0.1.1
+* Updated Automapper 6.2.0 -> 6.2.2 
+* Bug fix in Get-AzPeeringLocation 
```

Comment:
@rileymckenna what was the bug fix to this cmdlet?

Reply:
so my changes are definitely in my code, i must have slipped them in the first PR before it got merged.